test/src/xquery/xquery3/fnSort.xql

```diff
@@ -0,0 +1,172 @@
+xquery version "3.1";
+
+module namespace testSort="http://exist-db.org/xquery/test/function_sort";
+
+declare namespace test="http://exist-db.org/xquery/xqsuite";
+
+declare
+    %test:assertEquals("1","3","4","5","6")
+function testSort:single_parameter() {
+    fn:sort((1, 4, 6, 5, 3))
```

Comment:
But if the input is an integer, shouldn't the output be the same type? Why would the test expect the result to be of a different type?